google-cloud-bigquery/lib/google/cloud/bigquery/service.rb

```diff
@@ -473,6 +473,7 @@ def query_config query, options = {}
         def resolve_legacy_sql legacy_sql, standard_sql
```

Comment:
Is there a better place for this function? It is repeated across multiple files.